### PR TITLE
fix: fix issue with ButtonIcon inline issue

### PR DESCRIPTION
**Pull request by orrgottlieb**
Fix an issue with the wrapper of the icon button doesn't calculate the size of the content correctly

### CODE REVIEW

File: src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.scss [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 .monday-style-color-item-wrapper {
   --border-radius-multiplier: 1;
 
   &.circle {
     --border-radius-multiplier: 99999;
   }
-
+  /* stylelint-disable scss/operator-no-unspaced */
   @mixin absolute-border($border-width, $border-color-var-name, $inset: -$border-width, $border-radius: 7px) {
```
Comment (laviomri) [inline src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.scss:11]: why did you need the disable - was there a problem with `$inset: -$border-width`? 
if so, should we fix instead of disabling?

**Review by laviomri — APPROVED** [review]

File: src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.scss (hunk 1 above)
Comment (orrgottlieb) [reply src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.scss:11]: @laviomri will fix it after merge,
this branch should have no "potential" change
